subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/cell.go

```diff
@@ -183,8 +208,20 @@ func (c *PhysicalCell) GetVirtualCell() *VirtualCell {
 	return c.virtualCell
 }
 
-func (c *PhysicalCell) SetVirtualCell(vc *VirtualCell) {
-	c.virtualCell = vc
+func (c *PhysicalCell) SetVirtualCell(virtual *VirtualCell) {
```

Comment:
>virtual [](start = 38, length = 7)

Just name as "cell"? And align with func (c *VirtualCell) SetPhysicalCell(pc *PhysicalCell) {